### PR TITLE
bzip2: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by iMichka**
This is an automated pull request to build a new bottle for linuxbrew-core
based on the existing bottle block from homebrew-core.

### CODE REVIEW

**Comment by iMichka** [comment]
```
==> make install PREFIX=/home/linuxbrew/.linuxbrew/Cellar/bzip2/1.0.7
.
==> make -f Makefile-libbz2_so clean
.
==> make -f Makefile-libbz2_so
.
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - libbz2.so.1.0.6
```

**Comment by jonchang** [comment]
https://github.com/Homebrew/linuxbrew-core/issues/14121

File: Formula/bzip2.rb [hunk 1]
```diff
@@ -22,9 +22,9 @@ def install
       # Install the shared library.
       system "make", "-f", "Makefile-libbz2_so", "clean"
       system "make", "-f", "Makefile-libbz2_so"
-      lib.install "libbz2.so.1.0.6", "libbz2.so.1.0"
-      lib.install_symlink "libbz2.so.1.0.6" => "libbz2.so.1"
-      lib.install_symlink "libbz2.so.1.0.6" => "libbz2.so"
+      lib.install "libbz2.so.1.0.7", "libbz2.so.1.0"
```
Comment (jonchang) [inline Formula/bzip2.rb]:
```suggestion
      lib.install "libbz2.so.#{version}", "libbz2.so.1.0"
```

File: Formula/bzip2.rb [hunk 2]
```diff
@@ -22,9 +22,9 @@ def install
...
+      lib.install_symlink "libbz2.so.1.0.7" => "libbz2.so.1"
```
Comment (jonchang) [inline Formula/bzip2.rb]:
```suggestion
      lib.install_symlink "libbz2.so.#{version}" => "libbz2.so.1"
```

File: Formula/bzip2.rb [hunk 3]
```diff
@@ -22,9 +22,9 @@ def install
...
+      lib.install_symlink "libbz2.so.1.0.7" => "libbz2.so"
```
Comment (jonchang) [inline Formula/bzip2.rb]:
```suggestion
      lib.install_symlink "libbz2.so.#{version}" => "libbz2.so"
```

**Comment by issyl0** [comment]
Thanks for your suggestions, @joncheng. They look sensible, but I'm loathe to push more changes and have to see CI rebuild from the beginning. Once it's green in one run, maybe we can make cosmetic changes in another PR?

**Comment by issyl0** [comment]
`Build timed out after 5h0m0s`

**Comment by issyl0** [comment]
Azure has timed out too. My laptop is still ticking along nicely. These changes to `bzip2` themselves work fine, so in the interests of user experience, can we get this out quicker than waiting for `brew test-bot` to test all the dependencies?

Should we merge this without a bottle in order to fix the build, and leave it to build from source for a bit? Then later, we can work out how to do a bottle build and push (without CI, because CI will keep timing out)? (cc @sjackman)

**Comment by issyl0** [comment]
Merging this to fix the build. It won't have a bottle, but we can cross that bridge later. We'll at least fix people's `brew update`. :-) (Cleared with @sjackman.)

**Comment by sjackman** [comment]
Issy has built a bottle in Docker and published it.

**Pull request merged**